src/util/verify-chromium-port.js

```diff
@@ -0,0 +1,52 @@
+import { createServer } from 'node:http';
```

Comment:
```suggestion
import { createServer } from 'node:net';
```

We don't really need http capabilities, `node:net` is enough and we don't need `node:http`.